### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joshlk**
Document default display columns. As discussed in #97

### CODE REVIEW

**Review by Stonesjtu — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -76,6 +76,17 @@ For older versions (python 2.7, <3.4), you can continue using gpustat v0.x.
...
 [gh-issue-66]: https://github.com/wookayin/gpustat/issues/66
 
+Default display
+---------------
+
+> [0] GeForce GTX Titan X | 77'C,  96 % | 11848 / 12287 MB | python/52046(11821M)
+
+- `[0]`: GPUindex (starts from 0) as PCI_BUS_ID
+- `GeForce GTX Titan X`: GPU name
+- `77'C`: Temperature
```
Comment (Stonesjtu) [inline README.md:86]: probably a typo here?  `°` not `'`

Comment (Stonesjtu) [reply README.md:86]: Seems the raw output uses `'`, so let's make it a future work.

**Comment by wookayin** [comment]
Isn't it obvious? Did we have to add this to README?

**Pull request merged**